packages/templates/src/getTemplates.js

```diff
@@ -62,6 +62,8 @@ async function getTemplates(imageBaseUrl) {
     'no-days-off',
     'experience-thailand',
     'sleep',
+    'ad-template-one',
+    'ad-template-two',
```

Comment:
We can remove all other templates, so that we only have ad templates here

Reply:
removed the other story templates raw json data, but I think it may need further clean up to remove stickers as well and update the tests accordingly.

cc: @sayedtaqui